### PR TITLE
Set up Ruby code style checher

### PR DESCRIPTION
**Pull request by anatoliikryvishyn**
Set up automatic check of guidelines of Ruby code style.

### CODE REVIEW

**Comment by anatoliikryvishyn** [comment]
@serhiir, please review. Here is the statistic of rubocop statistic.

I will deploy this on server, for testing, after some other pull request will be merged, in order to not create new server.

**Comment by serhiir** [comment]
@anatoliikryvishyn Okay. There is a need to rebase as well.

**Comment by anatoliikryvishyn** [comment]
@serhiir, rebased.

**Comment by serhiir** [comment]
@anatoliikryvishyn But there are changes from #20. Please rebase again.

File: config/locales/en.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ en:
     create_btn: "Create list"
     create_message: "Create the list and start adding a new ToDo items"
     no_list_notification: "No Todo lists were created yet, create your first one"
+    successfuly_created_message: "List was created successfully."
+    navigate: "Navigate"
```
Comment (serhiir) [inline config/locales/en.yml]: @anatoliikryvishyn Still see non-relevant changes.

Comment (anatoliikryvishyn) [inline config/locales/en.yml]: @serhiir, can I update?

Comment (serhiir) [inline config/locales/en.yml]: @anatoliikryvishyn Sure you can. I just reviewed but didn't test anything as there is no mention it's ready for testing/review.

Comment (anatoliikryvishyn) [inline config/locales/en.yml]: @serhiir, will update, and deploy on staging after dinner.

**Comment by anatoliikryvishyn** [comment]
@serhiir, please test [here](https://trouble-tada-interlink.herokuapp.com/users/sign_in) and review.

**Comment by serhiir** [comment]
@anatoliikryvishyn Will take a look next time.

File: app/controllers/items_controller.rb [hunk 2]
```diff
@@ -1,30 +1,26 @@
...
 
   def update
     item = Items::TodoItem.new(params[:id])
-
-
-    render status: 200, json: {listId: item.list_id,
-                               itemId: item.id,
-                               done: item.done?} if item.toggle!(item_update_params)
+    render status: 200, json: { listId: item.list_id, itemId: item.id, done: item.done?
+        } if item.toggle!(item_update_params)
```
Comment (serhiir) [inline app/controllers/items_controller.rb]: @anatoliikryvishyn I suppose the reason of reformatting was the length of the line, right?
I suggest to apply the following formatting for better clarity:
```ruby
if item.toggle!(item_update_params)
  result = { listId: item.list_id, itemId: item.id, done: item.done? }
  render status: 200, json: result
end
```

Another way to do reformatting, I suppose, is the next:
```ruby
render 
  status: 200, 
  json: { listId: item.list_id, itemId: item.id, done: item.done? } if item.toggle!(item_update_params)
```

As for me, I like the first option though.

Comment (anatoliikryvishyn) [inline app/controllers/items_controller.rb]: >@anatoliikryvishyn I suppose the reason of reformatting was the length of the line, right?

@serhiir, yes.

File: app/views/lists/index.html.haml [hunk 3]
```diff
@@ -9,6 +9,5 @@
     .lists
       - if @lists.empty?
         = render partial: "lists/no_lists_notification"
-      - else
       - @lists.each do |list|
```
Comment (serhiir) [inline app/views/lists/index.html.haml]: @anatoliikryvishyn Can you please truncate all lists for sergeyr@interlink-ua.com on https://trouble-tada-interlink.herokuapp.com server so I could make sure this change breaks nothing.

Comment (anatoliikryvishyn) [inline app/views/lists/index.html.haml]: @serhiir, can i do this now, or after review?

Comment (anatoliikryvishyn) [inline app/views/lists/index.html.haml]: @serhiir, done.

**Comment by serhiir** [comment]
@anatoliikryvishyn Done.

**Comment by anatoliikryvishyn** [comment]
@serhiir, updated.

**Pull request merged**